common/src/main/scala/org/apache/comet/CometConf.scala

```diff
@@ -77,6 +77,15 @@ object CometConf extends ShimCometConf {
     .booleanConf
     .createWithDefault(true)
 
+  val COMET_FULL_NATIVE_SCAN_ENABLED: ConfigEntry[Boolean] = conf(
+    "spark.comet.native.scan.enabled")
+    .doc(
+      "Whether to enable the fully native scan. When this is turned on, Spark will use Comet to " +
+        "read supported data sources (currently only Parquet is supported natively)." +
+        " By default, this config is true.")
+    .booleanConf
+    .createWithDefault(true)
```

Comment:
I added a similar config locally. 🙂

Reply:
If it was was something related, you can merge it here. This is an experimental branch. If it is for something different, then we can change our name.

Reply:
No, I meant I tried to do similar native scan change into this branch. I was trying to enable DataFusion native Parquet scan yesterday. I found this branch this morning so tried to apply onto this branch.

I will see if any difference can be merged.